languages/ruby/exercises/concept/exceptions/.docs/introduction.md

```diff
@@ -0,0 +1,73 @@
+Exceptions in Ruby as in most languages provide a way of dealing with unexpected events. Proper handling of exceptions is important when trying to prevent your program from crashing.
```

Comment:
```suggestion
Exceptions in Ruby, as in many languages, provide a way of dealing with unexpected events. Proper handling of exceptions is important when trying to prevent your program from crashing.
```

More than 600 languages that I am aware of, but I have not quantified that most do this, probably safer, and less time sensitive, to state "many" rather than "most".

I might also change "unexpected events" to "exceptional events" since many events may be expected to happen that might still raise an exception.  Those exceptional events are the ones that we can manage.  The unexpected ones, we probably don't ever expect.